src/main/java/com/github/wovnio/wovnjava/Settings.java

```diff
@@ -79,7 +78,6 @@
 
         String defaultApiUrl = this.devMode ? DefaultApiUrlDevelopment : DefaultApiUrlProduction;
         this.apiUrl = stringOrDefault(reader.getStringParameter("apiUrl"), defaultApiUrl);
-        this.snippetUrl = this.devMode ? DefaultSnippetUrlDevelopment : DefaultSnippetUrlProduction;
```

Comment:
I think you want to preserve this behavior

Reply:
I'm not sure it's _that_ useful but let's get it back.